src/SiteMaster/Core/Auditor/Site/Reviews/Unfinished.php

```diff
@@ -0,0 +1,28 @@
+<?php
+namespace SiteMaster\Core\Auditor\Site\Reviews;
+
+use SiteMaster\Core\InvalidArgumentException;
+
+class Unfinished extends All
+{
+    public function __construct(array $options = array())
+    {
+        parent::__construct($options);
+    }
+
+    public function getSQL()
+    {
+        //Build the list
+        $sql = "SELECT id as id
+                FROM site_reviews
+                WHERE site_reviews.status != 'REVIEW_FINISHED'
```

Comment:
Fixed in https://github.com/mfairchild365/site_master/commit/60f164b81b9c4957bafb56a42527874e7ed25a3b